### ISSUE TITLE
DeprecationWarning: pkg_resources is deprecated as an API

### ISSUE THREAD

**Issue by agsimmons** [theatlantic/python-monkey-business#2]
When using this library on Python 3.11, the following warning is emitted:

```
lib/python3.11/site-packages/monkeybiz/__init__.py:3: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
```

https://setuptools.pypa.io/en/latest/pkg_resources.html

**Comment by tari** [comment]
This issue becomes more important with Python 3.12 because `setuptools` is [no longer installed by default](https://github.com/python/cpython/issues/95299). The better solution to address the deprecation warning is to use `importlib.resources` (supported since Python 3.7), but a bandaid that can also be applied to projects depending on this library is to manually add a dependency on `setuptools`.

**Comment by adamchainz** [comment]
Fixed in fbd3220132647b63ddbe5702179d392607a01090

**Comment by fdintino** [comment]
Oh, right. Thanks for reminding me to close this issue.
